satellite/overlay/service.go

```diff
@@ -148,6 +148,7 @@ type NodeDossier struct {
 	Contained    bool
 	Disqualified *time.Time
 	PieceCount   int64
+	ExitStatus   ExitStatusRequest
```

Comment:
nit- I wonder if we should just call this ExitStatus. Seems a little weird to return a `Request`. I actually created `ExitStatus` in another PR I'm working on and was going to see how this went over.

Reply:
You can remove, I can tweak in my PR after you merge in master.  Your call.

Reply:
I'll remove